docs/src/js/main.js

```diff
@@ -373,3 +373,12 @@ document.querySelectorAll('.switch:not(.switch--form)').forEach(function(widgetE
 
     widgetEl.addEventListener('makeup-switch-toggle', logEvent);
 });
+
+// TOGGLE-BUTTON
+document.querySelectorAll('.toggle-button').forEach(function(elToggleButton) {
+    elToggleButton.addEventListener("click", function() {
```

Comment:
This is personal preference, but I tend to view things like `aria-pressed="false"` as unnecessary. I'd rather have the aria role removed instead of set to the string "false":

```js
if (this.getAttribute('aria-pressed') === 'true') {
    this.removeAttribute('aria-pressed');
} else {
    this.setAttribute('aria-pressed', 'true');
}
```

If you go with this approach, the attribute for the component also won't be necessary when the button is not active so you don't need to include it in many of your examples.

No worries if you think your approach is better though, I don't mind if this isn't changed.

Reply:
Yea, I was trying to be more explicit, but I like the less markup approach better. @ianmcburnie , there are no screen reader performance benefits to having the attribute present all the time vs removing them, are there?

Reply:
For certain roles there are benefits in having an aria-attribute present as "false", but not in this case.

I personally go back and forth on my own preference on explicit vs non-explicit in cases like this :-) In this case I think I would lean more towards being explicit as it makes it easier to know every button is an ARIA toggle button.

Reply:
Ok, then let's keep them in the Skin docs as os. On the `ebay-ui` side though, @LuLaValva , it might make sense to do what you were suggesting since the markup is handled by the component and is not as important for _that_ implementation.